src/ResourceGraph/ResourceGraph.Test/ScenarioTests/ResourceGraphQueryTests.cs

```diff
@@ -50,7 +50,14 @@ public void Subscriptions()
         {
             TestController.NewInstance.RunPowerShellTest(_logger, "Search-AzureRmGraph-Subscriptions");
         }
-        
+
+        [Fact(Skip = "Needs investigation, TestManagementClientHelper class wasn't initialized with the ResourceManagementClient client.")]
```

Comment:
@OlhaTkachenko this error means that the `ResourceManagementClient` initialized in your `TestController` class is different (either by the package it comes from or the package version) than the `ResourceManagementClient` initialized in your cmdlet code. With the recent changes you made, I'm not sure if you're using the `ResourceManagementClient` anymore in your cmdlet code, so if you aren't, you should be able to run this test again.

Reply:
done